### ISSUE TITLE
Visual diff on the preview window in VS Code

### ISSUE THREAD

**Issue by jorgegarciamule** [mulesoft-labs/data-weave-language-server#45]
It would be great to provide a way of injecting an example of the output while using the preview on VS Code's plugin so the developer can quickly verify if the output matches what is expected.

**Comment by machaval** [comment]
Yes We have this feature on the IJ plugin and is very useful. Agree this is a very nice thing and useful +1
